metrics/sinks/influxdb/influxdb.go

```diff
@@ -197,14 +197,20 @@ func (sink *influxdbSink) createDatabase() error {
 	if sink.dbExists {
 		return nil
 	}
-	q := influxdb.Query{
+	qCreate := influxdb.Query{
 		Command: fmt.Sprintf("CREATE DATABASE %s", sink.c.DbName),
 	}
-	if resp, err := sink.client.Query(q); err != nil {
+	if resp, err := sink.client.Query(qCreate); err != nil {
 		if !(resp != nil && resp.Err != nil && strings.Contains(resp.Err.Error(), "already exists")) {
 			return fmt.Errorf("Database creation failed: %v", err)
 		}
 	}
+	qRetPol := influxdb.Query{
```

Comment:
if no retention policy is set in the sink config, should we still run this query?